rosa_architecture/rosa_policy_service_definition/rosa-service-definition.adoc

```diff
@@ -17,10 +17,14 @@ include::modules/rosa-sdpolicy-am-billing.adoc[leveloffset=+2]
 include::modules/rosa-sdpolicy-am-cluster-self-service.adoc[leveloffset=+2]
 include::modules/rosa-sdpolicy-am-compute.adoc[leveloffset=+2]
 include::snippets/pid-limits.adoc[]
+
+[role="_additional-resources-pid-limits"]
```

Comment:
According to the [_Additional resources_ guidance in our repo guidelines,](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections) the attribute should just be `[role="_additional-resources"]`.